### PR TITLE
add elb

### LINKED ISSUE #11: nifcloud_elb: Add resource
### Overview

Add a new resource `nifcloud_elb`.

### PR DESCRIPTION
**Pull request by fuku2014**
## Description

- Add nifcloud_elb resource.
- Close #11 

## How Has This Been Tested?

- [ ]  Buld with `make install` command, and Apply example.

```:console
make install
cd examples/elb
terraform init
terraform plan
terraform apply
```

## Types of Changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (improves code quality but has no user-facing effect)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

### CODE REVIEW

File: nifcloud/resources/elb/expander.go [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+		InstancePort:            nifcloud.Int64(int64(d.Get("instance_port").(int))),
+		Protocol:                computing.ProtocolOfNiftyConfigureElasticLoadBalancerHealthCheckRequest(d.Get("protocol").(string)),
+		HealthCheck: &computing.RequestHealthCheckOfNiftyConfigureElasticLoadBalancerHealthCheck{
+			Target:             nifcloud.String(d.Get("health_check_target").(string)),
+			Interval:           nifcloud.Int64(int64(d.Get("health_check_interval").(int))),
+			UnhealthyThreshold: nifcloud.Int64(int64(d.Get("unhealthy_threshold").(int))),
+		},
+	}
+
+	if strings.HasPrefix(nifcloud.StringValue(input.HealthCheck.Target), "HTTP") {
```
[the remark this answers is not in the document]
Comment (fuku2014) [reply nifcloud/resources/elb/expander.go:127]: YES!

File: nifcloud/resources/elb/expander.go [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+				},
+			},
+		},
+		AvailabilityZones: []string{d.Get("availability_zone").(string)},
+		NetworkVolume:     nifcloud.Int64(int64(d.Get("network_volume").(int))),
+		AccountingType:    computing.AccountingTypeOfNiftyCreateElasticLoadBalancerRequest(d.Get("accounting_type").(string)),
+		NetworkInterface:  networkInterface,
+	}
+
+	if strings.HasPrefix(nifcloud.StringValue(input.Listeners[0].RequestHealthCheck.Target), "HTTP") {
```
[the remark this answers is not in the document]
Comment (fuku2014) [reply nifcloud/resources/elb/expander.go:74]: YES!

File: nifcloud/resources/elb/schema.go [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+	}
+}
+
+func newSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		"elb_name": {
+			Type:        schema.TypeString,
+			Description: "The name for the multi load balancer.",
+			Optional:    true,
+			Computed:    true,
```
[the remark this answers is not in the document]
Comment (fuku2014) [reply nifcloud/resources/elb/schema.go:51]: The ELB name is optional and will be a random character string if omitted, so it is Computed.

[the remark this answers is not in the document]
Comment (tunakyonn) [reply nifcloud/resources/elb/schema.go:51]: OK.

File: nifcloud/resources/elb/expander.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (tunakyonn) [reply nifcloud/resources/elb/expander.go:74]: OK.

File: nifcloud/resources/elb/expander.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (tunakyonn) [reply nifcloud/resources/elb/expander.go:127]: OK.

**Review by tunakyonn — APPROVED** [review]
LGTM!

**Pull request merged**